### DISCUSSION TITLE
flight software project name

### DISCUSSION THREAD

**Discussion by capsulecorplab** [ODU-CGA-CubeSat/sealion-mission-architecture#36] [Ideas]
Referencing _ODU/CGA CubeSat Flight Software_ everywhere in the documentation can get cumbersome. Would be useful to come up with a shorter name for the flight software

**Comment by capsulecorplab** [comment]
name ideas for CubeSat:
- DeCoSat
- DiLoSat
- DeCoVIR-y

name ideas for flight software:
- DilophOS (short for Dilophosaurus Operating System)

**Reply by paclark5** [reply]
DeCoVIR-y
